lib/Sema/CSGen.cpp

```diff
@@ -3733,34 +3738,11 @@ namespace {
 
       if (auto closure = dyn_cast<ClosureExpr>(expr)) {
         if (closure->hasSingleExpressionBody()) {
-          CG.exitClosure(closure);
```

Comment:
Do we need the single-expression closure special case at all?

Reply:
Ah you are right, we do not need that, both single- and multi-statement closures should behave the same.